tests/UniswapAnchoredViewTest.js

```diff
@@ -468,8 +468,8 @@ describe('UniswapAnchoredView', () => {
       expect(await call(uniswapAnchoredView, 'lowerBoundAnchorRatio')).numEquals(new BigNumber(1e18).minus(anchorMantissa));
 
       await Promise.all(tokenConfigs.map(async config => {
-        const oldObservation = await call(uniswapAnchoredView, 'oldObservations', [config.uniswapMarket]);
-        const newObservation = await call(uniswapAnchoredView, 'newObservations', [config.uniswapMarket]);
+        const oldObservation = await call(uniswapAnchoredView, 'oldObservations', [config.symbolHash]);
```

Comment:
How did this work before?

Reply:
It was out of pity bc it was expecting 0...